src/meta/src/hummock/manager/mod.rs

```diff
@@ -514,6 +514,7 @@ async fn write_exclusive_cluster_id(
     const CLUSTER_ID_NAME: &str = "0";
     let cluster_id_dir = format!("{}/{}/", state_store_dir, CLUSTER_ID_DIR);
     let cluster_id_full_path = format!("{}{}", cluster_id_dir, CLUSTER_ID_NAME);
+    tracing::info!("try reading cluster_id");
```

Comment:
This is actually the slowest part now. We `stat` 2 non-exist objects, which takes 2s https://github.com/apache/opendal/discussions/5569